index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Styles Conference</title>
+    <link rel="stylesheet" href="stylesheets/reset.css" />
+    <link rel="stylesheet" href="stylesheets/style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@200&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <main class="main-div">
+      <div class="bg-image">
+        <header class="primary-header container">
+          <nav class="nav primary-nav container">
+            <div>
+              <button type="button" class="btn-apple">
+                <!--Apple-->
+                <img
+                  src="images/apple-logo-png.png"
+                  alt="apple-logo"
+                />&nbsp;</button
+              ><!--
+            --><button type="button" class="btn-apple">Store</button
+              ><!--
+            --><button type="button" class="btn-apple">Mac</button
+              ><!--
+            --><button type="button" class="btn-apple">iPod</button
+              ><!--
+            --><button type="button" class="btn-apple">iPhone</button
+              ><!--
+            --><button type="button" class="btn-apple">iPad</button
+              ><!--6.5
+            --><button type="button" class="btn-apple">iTunes</button
+              ><!--
+            --><button type="button" class="btn-apple">Support</button
+              ><!--
```

Comment:
- [ ] Here, the elements should be links ( `<a>` tag) that have `href` attributes linking to other sections of the page or just a '#'. I would also encourage you to make use of suitable semantic HTML elements like `<li>` instead of the `<button>` tag like you have it. It will make your codes look cleaner and modular.